modules/machineconfig-garbage-collect-viewing.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineconfig-garbage-collect-viewing_{context}"]
+= Viewing rendered machine configs
+
+You can view a list of rendered machine configs by using the `oc adm prune renderedmachineconfigs` command with the `list` subcommand. For example, the command in the following procedure would list all rendered machine configs for the `worker` machine config pool:   
+
+.Procedure
+
+* List the rendered machine configs:
++
+[source,terminal]
+----
+$ oc adm prune renderedmachineconfigs list --in-use=false --pool-name=worker
+----
++
+.Example output
+[source,terminal]
+----
+worker
+status: rendered-worker-ae115e2b5e6ae05e0e6e5d62c7d0dd81
+spec: rendered-worker-ae115e2b5e6ae05e0e6e5d62c7d0dd81
+----
++
+--
+where:
+
+list:: Displays a list of rendered machine configs in your cluster.
```

Comment:
```suggestion
`--list`:: Displays a list of rendered machine configs in your cluster.
```

Reply:
list is a subcommand and not a flag, so it does not need the '--' before it

Reply:
oopsie!